modules/nf-core/modules/fgbio/fastqtobam/main.nf

```diff
@@ -23,7 +23,7 @@ process FGBIO_FASTQTOBAM {
     def prefix = task.ext.prefix ?: "${meta.id}"
     """
     mkdir tmp
-
+    echo "fgbio --tmp-dir=${PWD}/tmp FastqToBam -i $reads -o ${prefix}_umi_converted.bam --read-structures $read_structure --sample $meta.id --library $meta.id $args" > /home/owacker/git/sarek/müde
```

Comment:
you know that you can see the actual command line used in the work folder in `.command.sh`

Reply:
yep I know. I think originally I printed out some other things as well and then just added this one